drivers/flash/Kconfig.nrf

```diff
@@ -33,4 +33,13 @@ config SOC_FLASH_NRF_UICR
 	  Enable operations on UICR. Once enabled UICR are written or read as
 	  ordinary flash memory. Erase is possible for whole UICR at once.
 
+config EMULATE_ONE_BYTE_WRITE_ACCESS
```

Comment:
@anangl @ioannisg @carlescufi This is proposal for making possible to use native write-block-size of nRF flash instead of byte access. It is according to my recommendation. If you are agree on this change (it is backward compatible) then @adamkondraciuk might continue with the patch.